addons/notes/src/Panel.tsx

```diff
@@ -10,12 +10,13 @@ import Markdown from 'markdown-to-jsx';
 
 import { PARAM_KEY, API, Parameters } from './shared';
 
-const Panel = styled.div({
+const Panel = styled.div(({theme}) => {
```

Comment:
Typescript is not compiling because a `return` statement or parenthesis around your custom theme are missing:
```
...({theme}) => ({
   padding: '3rem 40px',
   ...
   ...theme.addonNotesTheme
})
);